.github/workflows/daily-build.yml

```diff
@@ -22,13 +22,13 @@ jobs:
             - name: Download ballerina distribution
               uses: wei/wget@v1
               with:
-                args: http://dist-dev.ballerina.io/downloads/swan-lake-alpha5/ballerina-swan-lake-alpha5.zip
+                args: https://github.com/ballerina-platform/ballerina-distribution/releases/download/v2201.1.0/ballerina-2201.1.0-swan-lake.zip
```

Comment:
Shoudn't we get the daily built distribution or the latest distribution for the daily build?

Reply:
Yes, +1